src/jwtMiddleware.ts

```diff
@@ -87,54 +94,68 @@ export const jwtMiddleware = <
   T extends RouterMiddleware | Middleware = Middleware,
 >({
   key,
+  secret,
   algorithm,
-  critHandlers,
   customMessages = {},
   ignorePatterns,
-  onSuccess,
-  onFailure,
+  onSuccess = () => {},
+  onFailure = () => true,
 }: JwtMiddlewareOptions): T => {
+  if (typeof key === undefined) {
+    console.warn(
+      `oak-middleware-jwt: Property 'key' is going to be deprecated, you should now use 'secret' instead.`,
+    );
+  }
```

Comment:
Also added this since this is breaking, might not make sense because the type itself explains this 🤷‍♂️

Reply:
we can skip this as long as we have the deprecated notice for the type